extensions/answer_summarizers/models.py

```diff
@@ -90,6 +96,52 @@ def _get_top_answers_by_frequency(answers, limit=None):
     ])
 
 
+def _get_top_unresolved_answers_by_frequency(
+        answers_with_classification, limit=None):
+    hashable_answers_with_classification = [{
+        'hashable_answer': _HashableAnswer(ans['answer']),
+        'classification_categorization': ans['classification_categorization']
+    } for ans in answers_with_classification]
+
+    classification_results_dict = {}
+    for ans in hashable_answers_with_classification:
+        frequency = 0
+        if ans['hashable_answer'] in classification_results_dict:
+            frequency = classification_results_dict[ans['hashable_answer']][
+                'frequency']
+        classification_results_dict[ans['hashable_answer']] = {
+            'classification_categorization': (
+                ans['classification_categorization']),
+            'frequency': frequency + 1
+        }
+
+    answers_with_most_recent_classification = [{
```

Comment:
Could you explain the use of "recent" in the variable name? I don't see anything thus far in the function I'm reading that is time-dependent or pertains to recency.

Reply:
As I explained in above comment. I am keeping track of their most recent classification categorization from the list of the answers. Hence, the use of recent in the name as this list contains unique set of answers with their latest classification categorization.

Reply:
Ah ok, might be a documentation thing. See above.

Reply:
Done